File: src/main/kotlin/no/nav/arbeidsgiver/min_side/models/AltinnTilgangssøknad.java
package no.nav.arbeidsgiver.min_side.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
public class AltinnTilgangssøknad {
    public String orgnr;
    public String serviceCode;
    public Integer serviceEdition;
    public String status;
    public String createdDateTime;
    public String lastChangedDateTime;
    public String submitUrl;
}

File: src/main/kotlin/no/nav/arbeidsgiver/min_side/models/AltinnTilgangssøknad.kt
package no.nav.arbeidsgiver.min_side.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
data class AltinnTilgangssøknad(
    var orgnr: String? = null,
    var serviceCode: String? = null,
    var serviceEdition: Int? = null,
    var status: String? = null,
    var createdDateTime: String? = null,
    var lastChangedDateTime: String? = null,
    var submitUrl: String? = null
)